### ISSUE TITLE
The ability to prevent certain words from being submitted in form

### ISSUE THREAD

**Issue by Aryehhirsc** [elementor/elementor#15730]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I believe this feature will help with filtering out spam form submissions.

**Describe the solution you'd like**
The solution is allowing devs to ban certain words from being submitted in the comment section of a form. We get many forms offering us marketing, SEO, and other random services, but all these spam submissions usually have common phrases that I wouldn't mind banning.  

For Example, banning the word "SEO" would prevent about three spam submissions a day, not to mention all the inappropriate forms linking to porn websites & singles in the area!

**Comment by Aviad-Herman** [comment]
Hi,
did you try implementing the built-in reCaptcha and/or Honeypot fields?
https://elementor.com/help/form-fields/

If these don't work, please let me know and re-open this thread

**Comment by Aryehhirsc** [comment]
@Aviad-Herman Yes, I've tried Honeypots. I've even tried putting multiple honeypots in the same form, but it had minimal effect. As for reCaptcha, I've installed reCaptcha into my sites before, and I'm fine doing that on most sites; however, I can't afford the number of false positives on sites where I am doing google ads.

My post wasn't an attempt to get help on how to get rid of spam submissions; There are a thousand such posts. I am suggesting a method of getting rid of spam that does not cause false positives and shouldn't be that hard to implement.

**Comment by Aryehhirsc** [comment]
@Aviad-Herman How do I reopen the thread?

**Comment by Aviad-Herman** [comment]
Hey,
I re-opened it.
I will do some research on how many users requested this.

Thanks for bringing this up.

**Comment by Aryehhirsc** [comment]
@Aviad-Herman Thank you for looking into it. 

I feel like all the spam blocking available like reCaptcha, honeypot, or any other method all come with their issues, whether it be false positives or bots figuring out how to bypass it. This feature of blocking certain words gives the website owner the ability to fight the bots manually, so even if they figure out methods on how to bypass your blocks, the owner can go in and block the new method.

On one of my websites, there is a bot submitting forms using the word "p*rn' written with the "*" It would be nice to be able to go in and block that written like the way it is. 

Again thanks for looking into it. Would love to see this feature added. And if it is being added, I would recommend allowing users to download and upload negative lists so I can create one general word ban list and upload it to all my sites,
